treeherder/config/settings.py

```diff
@@ -177,41 +189,6 @@
     }
 }
 
-if DEBUG:
-    # TODO: Fold this into the logging config above, as part of bug 1318021.
-    LOGGING = {
-        'version': 1,
-        'disable_existing_loggers': True,
-        'formatters': {
-            'standard': {
-                'format': "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s",
-            },
-        },
-        'handlers': {
-            'console': {
-                'level': 'DEBUG',
```

Comment:
Do we need to add a conditional in 'handlers' for the debug level if ``DEBUG`` is true?

Reply:
I've left level configuration to the loggers, it always makes more sense to me to configure it next to where it gets used.